src/Functions.Worker.Extensions.OpenAI/Functions.Worker.Extensions.OpenAI.csproj

```diff
@@ -3,11 +3,18 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
     <Description>Microsoft Azure Functions .NET Isolated Worker Extension for OpenAI</Description>
+    <WebJobsVersion Condition="'$(WebJobsVersion)' == ''">$(VersionPrefix)-$(VersionSuffix)</WebJobsVersion>
   </PropertyGroup>
 
-
   <ItemGroup>
     <PackageReference Include="Azure.AI.OpenAI" Version="1.0.0-beta.13" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.3.0" />
   </ItemGroup>
+
+  <ItemGroup>
+    <AssemblyAttribute Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions.ExtensionInformationAttribute">
+      <_Parameter1>Microsoft.Azure.WebJobs.Extensions.OpenAI</_Parameter1>
+      <_Parameter2>$(WebJobsVersion)</_Parameter2>
+    </AssemblyAttribute>
```

Comment:
Cool! I didn't know we could do something like this! I have some other projects where this will be useful...